scripts/echo360.js

```diff
@@ -258,9 +263,9 @@ H5P.VideoEchoVideo = (() => {
         }
 
         const delta = Date.now() - this.lastTimeUpdate;
```

Comment:
This calculation is not correct if the playback rate is anything but 1. 

This results in the timeline running ahead and then jumping back if the playback rate is less than 0 (and will do the reverse for fast playback). 

delta needs to be multiplied by the playback rate.

Reply:
Indeed!